tree/tree/inc/ROOT/TIOFeatures.hxx

```diff
@@ -11,14 +11,21 @@
 #ifndef ROOT_TIO_FEATURES
 #define ROOT_TIO_FEATURES
 
-#include "TBasket.h"
-
-#include <initializer_list>
-#include <memory>
-#include <string>
-#include <sstream>
-#include <typeinfo>
-#include <type_traits> // is_same, enable_if
+#include "Rtypes.h"
+
+class TBasket;
+class TBranch;
+class TTree;
+
+// keep it here to have a note that was removed  
+// #ifndef R__LESS_INCLUDES
```

Comment:
That seems to contradict the removed `#include`s?

Reply:
This is again place, which Philippe recommend fully remove.
As intermediate solution, I keep this includes commented. 
If we face some problems with our users, we just have to uncomment this code